config/jetpack-cloud-production.json

```diff
@@ -36,8 +36,6 @@
 		"jetpack-cloud": true,
 		"jetpack-cloud/connect": false,
 		"jetpack/backups-date-picker": true,
```

Comment:
I see no reason why `"jetpack/scan-product"` and `"jetpack/anti-spam-product"` were not activated here as well.